ci/deps.lisp

```diff
@@ -0,0 +1,13 @@
+(ql-dist:install-dist "http://beta.quicklisp.org/dist/quicklisp/2018-01-31/distinfo.txt" :replace t :prompt nil)
```

Comment:
@echupriyanov This should not be done in the test environment, but rather be tested as part of the quickload process of each process.